pep-0008.txt

```diff
@@ -89,7 +89,7 @@ Yes::
     foo = long_function_name(var_one, var_two,
                              var_three, var_four)
 
-    # More indentation included to distinguish this from the rest.
+    # Added 4 spaces to distinguish arguments from the rest.
```

Comment:
I'm cool with this, although I might suggest a parenthetical, e.g.:

"Add 4 spaces (an extra level of indentation) to distinguish..."